node/src/protocol/state.rs

```diff
@@ -20,7 +21,10 @@ pub struct PersistentNodeData {
 }
 
 #[derive(Clone)]
-pub struct StartedState(pub Option<PersistentNodeData>);
+pub struct StartedState {
+    pub persistent_node_data: Option<PersistentNodeData>,
+    pub triple_data: Option<Vec<TripleData>>,
```

Comment:
why is this an option? I'd imagine this could just be an empty vec by default otherwise